.github/actions/bazel-test-all/action.yaml

```diff
@@ -121,3 +124,18 @@ runs:
             echo "Bazel version: $(bazel version)"
 
             bazel ${{ inputs.BAZEL_COMMAND }} "${bazel_args[@]}"
+
```

Comment:
Couldn't the "Write AWS credentials" step above be moved after the "Run Bazel Commands" step?

Alternatively, we could also integrate it with the "Upload to S3" step.